tests/examples/examples.yaml

```diff
@@ -1,5 +1,6 @@
 - path: "models"
   skip: ["choropleth.py", "maps_cities.py", "data_tables_server.py"]
+- path: "custom/gears"
```

Comment:
There must be added support for the following pattern: `examples/dir/example/__init__.py -> example.html`.